pkg/querier/tripperware/query.go

```diff
@@ -502,6 +504,16 @@ func BodyBufferFromHTTPGRPCResponse(res *httpgrpc.HTTPResponse, logger log.Logge
 	return res.Body, nil
 }
 
+func updateResponseSizeLimiter(res *http.Response, buf *bytes.Buffer, responseSizeLimiter *limiter.ResponseSizeLimiter) error {
```

Comment:
nit: Instead of calling this updateResponseSizeLimiter this should be getResponseSize(). Adding the size to limiter can be done in BodyBuffer() itself? 
```
s := getResponseSize()
responseSizeLimiter.AddResponseBytes(s)
```